### PR TITLE
Feature/feature/cpo 51 decouple frontend components from cms components 

### PR DESCRIPTION
**Pull request by Polleke007**

### CODE REVIEW

File: src/components/features/ctf-components/ctf-page/ctf-page-gql.tsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ const CtfPageGgl = ({ slug: slugFromProps }: Props) => {
     metaTags.no_follow === true ? 'nofollow' : undefined,
   ].filter((x): x is string => x !== undefined);
 
+  console.log("page", page);
```
Comment (ceneva) [inline src/components/features/ctf-components/ctf-page/ctf-page-gql.tsx:54]: Can this console.log be removed?

File: src/components/features/decoupled-components/hero-banner-component/hero-banner-component.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import {
+  DescriptionComponent
+} from '@src/components/features/decoupled-components/description-component/description-component';
+import {
+  ButtonComponent,
+  ButtonProps,
+} from '@src/components/features/decoupled-components/button-component/button-component';
+
+export const HeroBannerComponent = (props: ComponentHeroBannerFragment) => {
+  console.log("props", props);
```
Comment (ceneva) [inline src/components/features/decoupled-components/hero-banner-component/hero-banner-component.tsx:14]: Can this console.log be removed?

**Review by ceneva — APPROVED** [review]
Overal structure of the components seems to be way better :D maybe you can also insert the teaser component between the hero component and the smaller components (or later on). Maybe we can reflect on the folder structure for graphql, it would make more sense to me to group the graphql/contentful specific files in the same folder and not attached to the components.

File: src/components/features/ctf-components/ctf-expert-advice/ctf-expert-advice.tsx [hunk 3]
```diff
@@ -16,44 +16,29 @@ export default function CtfExpertAdvice({title, subtitle, buttonText1, buttonTex
...
-                  <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5}
-                       stroke="currentColor" className="w-6 h-6">
-                    <path strokeLinecap="round" strokeLinejoin="round"
-                          d="m12.75 15 3-3m0 0-3-3m3 3h-7.5M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z" />
-                  </svg>
-                </div>
-              </button>
+              {
+                advices && advices.items.map((item) => (
+                  <div key={item.title}>
```
Comment (ceneva) [inline src/components/features/ctf-components/ctf-expert-advice/ctf-expert-advice.tsx:21]: Wouldn't it make more sense to make a singlar advice component then iterate it with the data?

**Review by ceneva — APPROVED** [review]
Looks good overall, just a couple suggestions with some extra improvements.

File: src/components/features/ctf-components/ctf-expert-advice/ctf-expert-advice.tsx [hunk 4]
```diff
@@ -16,44 +16,29 @@ export default function CtfExpertAdvice({title, subtitle, buttonText1, buttonTex
...
+                    <a href={item.buttonUrl}>
+                      <button
```
[the remark this answers is not in the document]
Comment (Polleke007) [reply src/components/features/ctf-components/ctf-expert-advice/ctf-expert-advice.tsx:23]: We already have but I forgot to apply it to this component

**Pull request merged**